### DISCUSSION TITLE
Indoor camera with sound

### DISCUSSION THREAD

**Discussion by oguime** [blakeblackshear/frigate#2642] [General]
Hi,
Is anyone aware of an internal camera (wifi only) capable of recording sound on Frigate (RTSP/Onvif, h264 and AAC codec support) ? Thanks!

**Comment by oguime** [comment]
The Imou Cue 2 seems to have AAC/RTSP, and even substream support... Can anyone confirm if it will record sound and the auxiliary streams resolution?

**Reply by franpalacio** [reply]
Hola, yo me he comprado la imou cue 2c, tengo frigate en homeassistant OS, y no consigo que funcione, que configuración le pusiste?
Un saludo

**Reply by NickM-27** [reply]
Please make your own GitHub issue, it will make it easier to help
